tests/manage/pv_services/test_rwo_pvc_fencing_unfencing.py

```diff
@@ -0,0 +1,632 @@
+        worker_nodes_name, ocs_nodes_name, non_ocs_nodes_name = self.identify_nodes(
+            scenario, num_of_nodes
+        )
+
+        def finalizer():
+            if scenario == "colocated":
+                helpers.remove_label_from_worker_node(node_list=worker_nodes_name, label_key="nodetype")
+            else:
+                helpers.remove_label_from_worker_node(node_list=non_ocs_nodes_name, label_key="nodetype")
+
+        request.addfinalizer(finalizer)
+
+        project = project_factory()
+
+        # Select nodes for running app pods and inducing network failure later
+        ignore_check = (num_of_nodes == num_of_fail_nodes)
+        app_pod_nodes_name = self.select_nodes_for_app_pods(
+            scenario, ocs_nodes_name, non_ocs_nodes_name, num_of_fail_nodes,
+            ignore_check
+        )
+
+        # Create multiple RBD and CephFS backed PVCs with RWO accessmode
+        num_of_pvcs = num_of_app_pods_per_node * num_of_fail_nodes
+        rbd_pvcs = multi_pvc_factory(
+            interface=constants.CEPHBLOCKPOOL, project=project,
+            access_modes=[constants.ACCESS_MODE_RWO], num_of_pvc=num_of_pvcs,
+            wait_each=True
+        )
+        cephfs_pvcs = multi_pvc_factory(
+            interface=constants.CEPHFILESYSTEM, project=project,
+            access_modes=[constants.ACCESS_MODE_RWO], num_of_pvc=num_of_pvcs,
+            wait_each=True
+        )
+
+        # Create deploymentconfig based pods
+        dc_pods = []
+        # Start app-pods on selected node(s)
+        for node_name in app_pod_nodes_name:
+            helpers.label_worker_node(node_list=list({node_name}), label_key="nodetype", label_value="app-pod")
+            for num in range(num_of_app_pods_per_node):
+                dc_pods.append(
+                    dc_pod_factory(
+                        interface=constants.CEPHBLOCKPOOL, pvc=rbd_pvcs[0],
+                        node_selector={'nodetype': 'app-pod'}
+                    )
+                )
+                dc_pods.append(
+                    dc_pod_factory(
+                        interface=constants.CEPHFILESYSTEM, pvc=cephfs_pvcs[0],
+                        node_selector={'nodetype': 'app-pod'}
+                    )
+                )
+                rbd_pvcs.remove(rbd_pvcs[0])
+                cephfs_pvcs.remove(cephfs_pvcs[0])
+            helpers.remove_label_from_worker_node(node_list=list({node_name}), label_key="nodetype")
+
+        # Label all the nodes to be able to run app pods
+        if scenario == "colocated":
+            helpers.label_worker_node(node_list=ocs_nodes_name, label_key="nodetype", label_value="app-pod")
+        else:
+            helpers.label_worker_node(node_list=non_ocs_nodes_name, label_key="nodetype", label_value="app-pod")
+
+        # Get ceph mon,osd pods running on selected node(s) for colocated scenario
+        ceph_pods = []
+        if scenario == "colocated" and num_of_nodes > 3:
```

Comment:
Let's try avoid using an absolute number (3)

Reply:
Mon and OSD pod will try to move to other nodes and stuck in `Init` state only when there is extra OCS node present in the cluster, that's why checking for absolute value (3).